### DISCUSSION TITLE
[Feature] <Internationalization Support>

### DISCUSSION THREAD

**Discussion by kkaaddff** [immich-app/immich#8849] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I am from China, and I am using Immich to build an application for my family to use. I am particularly looking forward to internationalization support for the Web.
If possible, I can complete this feature. In fact, I am currently building this capability based on svelte-i18n.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1688. A PR would be welcome! We're currently using Localizely as the translation platform but we intend to move to Weblate, so if you can keep Weblate in mind if you build it that'd be awesome.
